runbotics/runbotics-orchestrator-ui/src/src-app/Actions/zip.actions.ts

```diff
@@ -56,6 +54,95 @@ const getZipActions: () => Record<string, IBpmnAction> = () => ({
                 }
             }
         }
+    },
+    [ZipAction.ZIP_FILE]: {
+        id: ZipAction.ZIP_FILE,
+        label: translate('Process.Details.Modeler.Actions.Zip.Zip.Label'),
+        script: ZipAction.ZIP_FILE,
+        runner: Runner.DESKTOP_SCRIPT,
+        output: {
+            assignVariables: true,
+            outputMethods: {
+                variableName: '${content.output[0]}',
+            },
+        },
+        form: {
+            schema: {
+                type: 'object',
+                properties: {
+                    input: {
+                        title: translate('Process.Details.Modeler.Actions.Common.Input'),
+                        type: 'object',
+                        properties: {
+                            fileName: {
+                                title: translate('Process.Details.Modeler.Actions.Zip.Zip.Name'),
```

Comment:
why Zip.Zip?
maybe Zip.Create or Zip.CreateArchive

Reply:
good idea. Changes "Unzip" to "Unzip archive" and "Zip" to "CreateArchive" to be consistent with UI